config/config.exs

```diff
@@ -1,9 +1,25 @@
 use Mix.Config
 
+config :erlang_node_discovery,
+  hosts: ["127.0.0.1", "crawlyui.com"],
+  node_ports: [{:ui, 0}]
+
+ui_node = System.get_env("UI_NODE") || "ui@127.0.0.1"
+ui_node = ui_node |> String.to_atom()
+
```

Comment:
This does not seem to be related to  the spider work itself.

Reply:
I would prefer to follow more link, comparing to the idea of querying all seen URLs, as it's expensive. I would imagine a full amazon crawl would require you to make 100M of requests at least.

Reply:
Hmm I see, yeah in that sense it would be better to follow more links. I guess I need to optimise more on how to define user agents and request to amazon so it won't block me before I get to the 100M requests haha